app/globals.css

```diff
@@ -1,3 +1,39 @@
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
\ No newline at end of file
+@tailwind utilities;
+
+.slick-track {
+  @apply rs-py-8 px-0;
+}
+
+.slick-slide > div {
+  @apply rs-mx-1;
+}
+
+@media (min-width: 992px) {
+  .center .slick-slide.slick-active.slick-current {
+    @apply z-50 transition-all duration-[400ms] ease-[ease-in-out] scale-[1.5] opacity-100 relative;
```

Comment:
@yvonnetangsu any thoughts or feedback on how the scaling of the center slide? 🤔 
The scale increase impacts the caption sizing as well.

Reply:
@rebeccahongsf I didn't look at the code so just answering theoretically. Is it possible to have a wrapper around the image and caption, then you scale the wrapper, and just have the image inside be full width of the wrapper?

Reply:
I almost feel like we should only show the caption when the slide is in the middle 🤔